tools/istio-docker.mk

```diff
@@ -79,9 +79,7 @@ docker.proxyv2: install/gcp/bootstrap/gcp_envoy_bootstrap.json
 docker.proxyv2: $(ISTIO_ENVOY_LINUX_RELEASE_DIR)/envoy
 docker.proxyv2: $(ISTIO_OUT_LINUX)/pilot-agent
 docker.proxyv2: pilot/docker/Dockerfile.proxyv2
-docker.proxyv2: pilot/docker/envoy_pilot.yaml.tmpl
```

Comment:
these are not strictly related to this change, but these are neither used nor do they work anymore. I cleaned them up since they reference the SDS token field I removed